documents-git/includes/config.json

```diff
@@ -1,5 +1,7 @@
 {
   "limit": 5,
+  "cache_strategy": "static",
```

Comment:
I figured `enable_cache` is not really necessary anymore since it works out-of-the-box with transients. I added this one to allow for your idea with webhooks for dynamic caching.